src/main/webapp/college-info.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+  </head>
+  <body>
+    <header>
+      <h1>Black Student Insight</h1>
+      <input type="text" id="school-search">
+      <search-button id="search-button">Search</search-button>
+    </header>
+    <main>
+      <h1 id="school-name"></h1>
+
+      <section>
+        <p id="school-desc"></p>
+        <p id="students"></p>
+      </section>
+
+      <section>
+        <p id="admissions"></p>
+        <p id="cost"></p>
+      </section>
+
+      <section>
+        <p id="race-piechart"></p>
+        <p id="gender-piechart"></p>
+      </section>
+
+      <a href="college-info.html">Back to Profile Home</a>
+        <h2>Student Reviews</h2>
+        <div id="comments-container"></div>
+        <h2>Leave a Review</h2>
+        <form id="submit-review" method="POST">
+          <input type="hidden" name="school-id" id="school-id">
+          <input name="name-input" placeholder="Name...">
+          <textarea name="text-input" placeholder="Message..."></textarea>
+          <button type="submit">Submit</button>
```

Comment:
In the comments.html file, I changed this line so that it could be styled. I would make sure the line is the same.

Reply:
didn't we get rid of comments.html

Reply:
Yeah, I was saying to make sure this line was the same before you removed the comments.html file.

Reply:
oh but add the change to this file..i got you